src/app-bundles/locations-map-bundle.js

```diff
@@ -60,6 +60,21 @@ export default {
 
   selectLocationsMapMapState: state => state.locationsMap.mapState,
 
+  /**
+   * Get Map query params but convert numeric params (lat/lon/zoom) from strings to numbers for later use
+   */
+  selectLocationMapQueryObject: createSelector(
```

Comment:
This is a nice idea but it doesn't work if the lat, lon, and zoom are empty strings. If the url is `lat=&lon=&zoom=` then the Map and MapDetailsHeader components throw a warning since they are expecting those props to be numbers, but instead they will be empty strings. You can see this happening if you go in the location tree and select North Atlantic -> Baltimore -> Allegheny River. The user can also manually change the url and accidentally remove the lat and lon themselves. I think we're better off using the selectQueryObject so that we know the props will always be a string regardless of what's in the url.